src/components/button/button.spec.ts

```diff
@@ -130,14 +130,32 @@ describe('MdButton', () => {
     });
 
   });
+
+  // Ripple tests.
+  describe('button ripples', () => {
+    it('should remove ripple if md-ripple-disabled input is set', (done: () => void) => {
+      return builder.createAsync(TestApp).then(fixture => {
+        let testComponent = fixture.debugElement.componentInstance;
+        let buttonDebugElement = fixture.debugElement.query(By.css('button'));
+
+        fixture.detectChanges();
+        expect(buttonDebugElement.nativeElement.querySelectorAll('[md-ripple]').length).toBe(1);
+
+        testComponent.rippleDisabled = true;
+        fixture.detectChanges();
+        expect(buttonDebugElement.nativeElement.querySelectorAll('[md-ripple]').length).toBe(0);
+        done();
+      });
+    });
+  });
```

Comment:
Rather than using `done()`, just make the entire test async:


```ts
it('should do a thing', async(() => {
  // ...
}));
```

It will automatically complete once all of the asynchronous tasks have completed.